src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+
+import usersFromServer from '../../api/users';
+import { TodoWithUser } from '../../Types/TodosWithUsers';
+import { getUserByUserId } from '../../utils/GetUserByUserId';
+import { biggestId } from '../../utils/BiggestId';
+
+interface Props {
+  onAddTodo: (newTodo: TodoWithUser) => void;
+  todos: TodoWithUser[];
+}
+
+export const TodoForm: React.FC<Props> = ({ onAddTodo, todos }) => {
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+
+  const [selectedUser, setSelectedUser] = useState(0);
```

Comment:
```suggestion
  const [selectedUserId, setSelectedUserId] = useState(0);
```
You store an id, not a user